packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -2212,6 +2213,402 @@ curl -v -X GET \
 ]
 ```
 
+#### List apps by name
+
+Enumerates all apps by the app name
+
+##### Request example: List apps with `name` eq `workday`
```

Comment:
The other examples seem to just use `Request Example`

Reply:
@luishernandezcruz-okta - Changed this back to `Request Example`. Please review.